lib/features.js

```diff
@@ -52,6 +53,23 @@ module.exports = config => {
     },
 
     /**
+     * Return whether an account meets the account created age requirement to
+     * bypass sign-in confirmation.
+     *
+     * @returns {boolean}
+     */
+    isSigninBypassAccountWithAgeEnabled(account) {
```

Comment:
Hrm, I've always thought of `features` as a thing that we are rolling out to subset of users that we eventually plan to promote to everyone, and then that 'feature' will be ripped out of here, such as what happened to signin confirmation.